### DISCUSSION TITLE
Request: Method to install mod to different instance

### DISCUSSION THREAD

**Discussion by setzke** [modrinth/.github#197] [Feature requests]
When inside an instance, viewing mods within that instance, it'd be convenient if one could right click in versions, or use a button, to "Install to other instance".

Presently if I wanted to manually copy mods over from one instance to another via the GUI, I'd need to go into the source instance, look at the list of mods, exit the instance, go search for those mods in the catalog, install to new instance, then go check the initial list again to repeat the tedious process for any additional mods I'd like to cherry pick.

**Comment by ardunster** [comment]
I was wishing for this feature today myself.
